sections/proof-of-yield/DailyYield.tsx

```diff
@@ -0,0 +1,156 @@
+import moment from "moment";
+const { commify } = utils;
+
+interface DailyYieldProps {
+  dailyStats: DailyStat[];
+}
+
+const viewMoreAmount = 30;
+
+const DailyYield = ({ dailyStats }: DailyYieldProps) => {
+  const width = useViewWidth();
+  const router = useRouter();
+  const [stats, setStats] = useState(dailyStats);
+
+  const routeToYieldOnDay = (date: number) => {
+    router.push(`/proof-of-yield/${date}`);
+  };
+
+  const moreDays = async (begin: number, end: number) => {
+    const moreStats = await fetchDailyStats(end, begin);
+    setStats(() => [...moreStats]);
+  };
+
+  return (
+    <Section className="mt-10 md:mt-28">
+      {/* Buttons */}
+      {/* <div className="mb-3 sm:mb-6 bg-tooltip w-fit p-2 rounded-[100px]">
+        <button
+          className={twMerge(
+            `rounded-[100px] border border-tooltip px-12 py-3`,
+            ` ${days ? highlightCss : ""}`
+          )}
+          onClick={() => seTableDataays(true)}
+        >
+          Days
+        </button>
+        <button
+          className={twMerge(
+            `rounded-[100px] border border-tooltip px-12 py-3`,
+            ` ${!days ? highlightCss : ""}`
+          )}
+          onClick={() => seTableDataays(false)}
+        >
+          Blocks
+        </button>
+      </div> */}
+
+      {/* Main Table */}
+      <Table>
+        {/* Table Head */}
+
+        <thead>
+          <tr>
+            <TableHead align="left" className="pl-8">
+              Date
+            </TableHead>
+            <TableHead
+              info={`The actual amount of OETH added to users' wallet balances`}
+              className="whitespace-normal sm:whitespace-nowrap pr-8 lg:pr-14 xl:pr-24"
+            >
+              Yield distributed
+            </TableHead>
+            <TableHead
+              info={`The annualized, compounded rate earned by OETH holders on this day`}
+              className="pr-8 sm:pr-8 lg:pr-14 xl:pr-24"
+            >
+              APY
+            </TableHead>
+            {width >= smSize && (
+              <TableHead
+                info={`The sum of all assets currently held by OETH`}
+                className="px-8"
+              >
+                Vault value
+              </TableHead>
+            )}
+            {/* {width >= smSize && (
+              <TableHead info={true} className="pr-0 xl:pr-8">
+                Vault value
+              </TableHead>
+            )} */}
+            {/* <TableHead></TableHead> */}
+          </tr>
+        </thead>
+
+        {/* Table Body */}
+
+        <tbody className="relative px-6">
+          {stats.map((item, i) => (
+            <tr
+              className="group border-t md:border-t-2 hover:bg-hover-bg border-origin-bg-black /cursor-pointer"
+              key={item.date}
+              // onClick={() => routeToYieldOnDay(item.date)}
+            >
+              <TableData align="left" className="pl-8">
```

Comment:
As Im looking at this usage, I dont think we should really even create a wrapper for `td` or `th`, might as well just add the elements directly

Reply:
The point of the wrappers is mostly to be able to not have to duplicate the styling. Initially, I used the `td` and `th` elements directly and just had `[element]ClassName` strings at the top of the file but we decided it was better to just make it into a component.